src/nrniv/nrnpy.cpp

```diff
@@ -272,15 +279,18 @@ static nrnpython_reg_real_t load_nrnpython() {
         auto const iter =
             std::find(supported_versions.begin(), supported_versions.end(), pyversion);
         if (iter == supported_versions.end()) {
-            std::cerr << "Python " << pyversion
-                      << " is not supported by this NEURON installation (supported:";
+            Fprintf(
+                stderr,
+                fmt::format("Python {} is not supported by this NEURON installation (supported:",
+                            pyversion)
+                    .c_str());
             for (auto const& good_ver: supported_versions) {
-                std::cerr << ' ' << good_ver;
+                Fprintf(stderr, fmt::format(" {}", good_ver).c_str());
```

Comment:
Still could do https://godbolt.org/z/e7WPabWTE

Reply:
Didn't see your response above, this is a non-blocking thread :)

Reply:
One change at a time.